cpp/example_code/s3/README.md

```diff
@@ -1,6 +1,6 @@
-# Amazon S3 C++ SDK code examples using S3Client
+#  Amazon S3 C++ SDK code examples using S3Client
 
-## Purpose
+## Overview
 The code examples in this directory demonstrate how to work with the Amazon Simple Storage Service 
 (Amazon S3) using the AWS SDK for C++.
```

Comment:
The code examples in this directory demonstrate how to work with Amazon Simple Storage Service 
(Amazon S3) by using the AWS SDK for C++.

Reply:
I have fixed this